### PR TITLE
docs: added ebl generic template portion

### PR DESCRIPTION
**Pull request by ryanseee**

### CODE REVIEW

File: docs/appendix/document-creator.md [hunk 1]
```diff
@@ -405,3 +405,72 @@ _Note: **Please replace all the placeholder "<...>" with your values.**_
...
 ---
+
+## Generic Template (Bill of Lading)
+
+We have designed and created a simple generic template (Bill of Lading) config file for you to use.
+
+Click <a href="/docs/appendix/document-creator/bill-of-lading/ebl-generic-template-config-file.json" target="_blank" rel="noopener noreferrer" download="ebl-generic-template-config-file.json">here</a> to download a config file.
+
+![Generic Template](/docs/appendix/document-creator/bill-of-lading/ebl-generic-template.png)
+
```
Comment (simboonlong) [inline docs/appendix/document-creator.md:416]:
```suggestion
![Generic Template - Bill of Lading](/docs/appendix/document-creator/bill-of-lading/ebl-generic-template.png)

```

also, able to not use govtech logo for ebl? maybe use the TT one will do

File: docs/appendix/document-creator.md [hunk 2]
```diff
@@ -384,7 +384,7 @@ _Note: **Please replace all the placeholder "<...>" with your values.**_
 ```
 
 - `"documentStore"` - Update with your document store. For more infomation, click [here](/docs/verifiable-document/document-store).
-- `"location"` - Update with the Issuer's domain. For more information, click [here](https://openattestation.com/docs/appendix/identity-proofs).
+- `"location"` - Update with the Issuer's domain. For more information, click [here](https://openattestation.com/docs/advanced/identity-proofs).
```
Comment (simboonlong) [inline docs/appendix/document-creator.md:387]: able to link to the local one instead?

File: static/docs/appendix/document-creator/bill-of-lading/ebl-generic-template-config-file.json [hunk 3]
```diff
@@ -0,0 +1,100 @@
+{
+  "network": "<Your network>",
```
Comment (simboonlong) [inline static/docs/appendix/document-creator/bill-of-lading/ebl-generic-template-config-file.json:2]: able to rename this json file? understand that there was 1 `generic-template.json` which is the cover letter one, u wanna update both names so they make sense each?

```
generic-template-ebl.json
generic-template-cover-letter.json
```
maybe something ^

File: docs/appendix/document-creator.md (hunk 1 above)
Comment (ryanseee) [reply docs/appendix/document-creator.md]: i have changed it already.

File: docs/appendix/document-creator.md (hunk 2 above)
Comment (ryanseee) [reply docs/appendix/document-creator.md]: i have changed it already.

File: docs/appendix/document-creator.md [hunk 4]
```diff
@@ -405,3 +405,72 @@ _Note: **Please replace all the placeholder "<...>" with your values.**_
...
+
+## Generic Template (Bill of Lading)
+
+We have designed and created a simple generic template (Bill of Lading) config file for you to use.
+
+Click <a href="/docs/appendix/document-creator/bill-of-lading/ebl-generic-template-config-file.json" target="_blank" rel="noopener noreferrer" download="ebl-generic-template-config-file.json">here</a> to download a config file.
+
+![Generic Template - Bill of Lading](/docs/appendix/document-creator/bill-of-lading/ebl-generic-template.png)
+
+Above is an example of how the bill of lading will look like.
```
Comment (simboonlong) [inline docs/appendix/document-creator.md:417]: generic bill of lading

File: docs/appendix/document-creator.md [hunk 5]
```diff
@@ -405,3 +405,72 @@ _Note: **Please replace all the placeholder "<...>" with your values.**_
...
+    "url": "https://generic-templates.tradetrust.io"
+  },
+  "issuers": [
+    {
+      "identityProof": {
+        "type": "DNS-TXT",
+        "location": "<Issuer's domain>"
+      },
+      "name": "DEMO STORE",
+      "tokenRegistry": "<Your token registry>"
```
Comment (simboonlong) [inline docs/appendix/document-creator.md:447]:
```suggestion
      "name": "DEMO TOKEN REGISTRY",
      "tokenRegistry": "<Your token registry>"
```

File: docs/appendix/document-creator.md (hunk 4 above)
Comment (ryanseee) [reply docs/appendix/document-creator.md]: amended.

File: docs/appendix/document-creator.md (hunk 5 above)
Comment (ryanseee) [reply docs/appendix/document-creator.md]: amended.

**Review by simboonlong — APPROVED** [review]
lgtm, rmb to choose squash and merge

Screenshot 2021-01-28 at 9 56 40 PM

**Comment by simboonlong** [comment]
oh ya choose squash and merge option already, also rmb to delete all the previous in between commit messages too

**Pull request merged**